packages/armkit-cli/lib/type-generator.ts

```diff
@@ -265,7 +265,7 @@ export class TypeGenerator {
     this.emitLater(typeName, code => {
       code.openBlock(`export enum ${typeName}`);
       values.forEach((v) => {
-        const validName = `${v}`.match(/^([a-zA-Z_])+$/) ? constantCase(`${v}`) : `"${constantCase(`${v}`)}"`
+        const validName = `${v}`.match(/^([a-zA-Z_])+$/) ? constantCase(`${v}`) : `"${typeName}_${constantCase(`${v}`)}"`
```

Comment:
this is what's causing the snapshot failures. The generated typescript legitimately does not match the saved versions from before. You can regenerate the snapshots with `yarn test -u`.